cmd/create/cluster/cmd.go

```diff
@@ -657,23 +660,27 @@ func init() {
 	)
 	flags.MarkHidden("billing-account")
 
+	flags.BoolVar(
+		&args.createAdminUser,
+		"create-admin-user",
+		false,
+		`Create cluster admin`,
+	)
+
 	flags.StringVar(
 		&args.clusterAdminUser,
 		"cluster-admin-user",
 		"",
-		`Username of Cluster Admin.
-Username must not contain /, :, or %%`,
+		`Deprecated cluster admin flag. Please use --create-admin-user.`,
 	)
-
 	flags.StringVar(
 		&args.clusterAdminPassword,
 		"cluster-admin-password",
 		"",
-		`Password of Cluster Admin.
-The password must
-- Be at least 14 characters (ASCII-standard) without whitespaces
-- Include uppercase letters, lowercase letters, and numbers or symbols (ASCII-standard characters only)`,
+		`Password of Cluster Admin.`,
```

Comment:
Should we provide details about the password requirements in the help message? 
Like we have done previously.

Reply:
Thanks for the find!